### ISSUE TITLE
Only "show method parameter names" for highlighted line

### ISSUE THREAD

**Issue by ChristianCiach** [angelozerr/jdt-codemining#52]
I love this plugin!

But I think I would like to have the option to only see the parameter names for the line the cursor is currently placed in (or maybe also the surrounding lines, if a method call spans multiple lines).

**Comment by angelozerr** [comment]
Indeed, it should be very cool. I had had this idea too to display the git author at the end of the line of the cursor. But the main problem is that code mining Platform Text doesn' support this feature. Just find time to do that.

**Comment by angelozerr** [comment]
I have created a bug https://bugs.eclipse.org/bugs/show_bug.cgi?id=541317 for this issue.
